Formula/ppsspp.rb

```diff
@@ -2,8 +2,8 @@ class Ppsspp < Formula
   desc "PlayStation Portable emulator"
   homepage "https://ppsspp.org/"
   url "https://github.com/hrydgard/ppsspp.git",
-      :tag      => "v1.10",
-      :revision => "401df203844465b84015d4710c6f02914b890589"
+      :tag      => "v1.10.1",
```

Comment:
this is not the latest release though
https://github.com/hrydgard/ppsspp/releases

Reply:
It seems they stopped marking minor version tags as releases starting with 1.7.2

Reply:
Could you ask if this is intentional?

Reply:
hrydgard/ppsspp#13101 (Also asked on their Discord server)

Reply:
From a moderator in the #coding_development channel:

> yes, but v1.10.2 is probably coming soonish (it mostly affects Android, but there's a fix that affects macOS too)